dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite."
+DESCRIPTION="It is for users and developers wanting to create databases, search, and edit data. It uses a familiar spreadsheet-like interface, and you don't need to learn complicated SQL commands.\
+Controls and wizards are available for users to:\
+Create and compact database files\
+Create, define, modify and delete tables\
+Create, define and delete indexes\
+Browse, edit, add and delete records\
+Search records\
+Import and export records as text\
+Import and export tables from/to CSV files\
+Import and export databases from/to SQL dump files\
+Issue SQL queries and inspect the results\
+Examine a log of all SQL commands issued by the application"
+HOMEPAGE="https://github.com/sqlitebrowser/sqlitebrowser"
+COPYRIGHT="Copyright notice>"
+LICENSE="GNU GPL v3"
+REVISION="1"
+scrGitRev="70ff691f1a8a82c91c22160c6d990fbee9bc0eb7"
+SOURCE_URI="$HOMEPAGE/archive/$scrGitRev.tar.gz"
+CHECKSUM_SHA256=""
+SOURCE_DIR="sqlitebrowser-$scrGitRev"
+
+ARCHITECTURES="x86_gcc2  x86"
+
+PROVIDES="
+    sqlitebrowser = $portVersion
+    libs:antlr-2.7.7
+    libs:qcustomplot-source
+    libs:qhexedit
+    libs:qscintilla
+    "
+REQUIRES="
+        haiku >= $haikuVersion
+        "
+BUILD_REQUIRES="
+	
+        "
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:cmake
+	cmd:qt-devel
+	cmd:ant-antlr
+	cmd:sqlite-devel
+	cmd:gcc_c++
```

Comment:
I read under some building guide in sqlitebrowser that "Make sure the cmake, qt-devel, ant-antlr, sqlite-devel, antlr-C++, and gcc-c++ packages are installed"
So I thought BUILD_PREREQUIRES would be the perfect location for this .

I am really sorry if I did something really stupid .